03-components.Rmd

```diff
@@ -1,10 +1,129 @@
+# Components of a reproducible analysis
 
 ```{r, include = FALSE}
 ottrpal::set_knitr_image_path()
 ```
 
+In this chapter, we will discuss what components of an analysis make it reproducible.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g21a84b32106_0_23")
+```
+
+## Reproducibility is iterative work
+
+Making an analysis isn't something that happens on the first try. Working on a project iteratively and continuing to improve the reproducibility of it is the best approach. In this manner, we can view reproducibility on a continuum. Some projects are just run once but aren't really needed anymore, and don't become very polished or reproducible. But as we continue to work on a project and polish its reproducible components, it continues to be more perfected. However, because of the moving nature of some reproducibility components, no project is really perfectly reproducible in every context throughout time. 
+
 ```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
 ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g21a84b32106_0_23")
 ```
+
+## Components of reproducibility
+
+A reproducible analysis is transparent, consistent, and accessible.
+
+- **Transparency** refers to the idea that it is well communicated and everything is displayed: data, code, goals, methods, and decisions. There are no secrets in a reproducible analysis/
+- **Consistency** refers to the idea that the code can be consistently run, but also everything follows a particular system, conventions and design.
+- **Accessibility** refers to the idea that anyone anywhere should be able to run and/or examine the analysis. No pay walls or expensive software should be required.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g21b8e34c516_0_5")
+```
+
+## Transparent
+
+Code that runs reliably is important but clear communication about the analysis and the decisions made make an analysis reproducible. A transparent analysis is not only well communicated, but also shared publicly in a way that others can comment and contribute ideas and suggestions to it or borrow the methods and strategies for their own analyses.
+
+### Open source
+
+Open source means not making code and data publicly available but that the analysis is able to be modified by anyone. This doesn't mean that any and all modifying contributions need to be accepted, because some level of standards and quality checks need to be maintained by the owners of the analysis, but just that anyone online could propose a contribution if they wanted to.
```

Comment:
```suggestion
Open source means not only making code and data publicly available, but also enabling others to modify or comment on the code. This doesn't mean that any and all modifying contributions need to be accepted, because some level of standards and quality checks need to be maintained by the owners of the analysis, but just that anyone online could propose a contribution if they wanted to.
```

Reply:
just making the sentence structure parallel

Reply:
also makes it seem like we are building on my definition of open source in the earlier chapter